src/main/proto/inter_node_rpc_service.proto

```diff
@@ -101,6 +101,9 @@ enum MetricEnum {
 
   // Heap
   HEAP_MAX = 16 [(additional_fields).name = "heap max", (additional_fields).description = "max heap size in bytes"];
+
+  // GC
+  FULL_GC = 32 [(additional_fields).name = "full gc", (additional_fields).description = "pause time in ms"];
```

Comment:
It seems `MINOR_GC` is described as `"time in percentage"`. It that valid?

Reply:
Changed the description